libimage/runtime.go

```diff
@@ -161,6 +161,15 @@ func (r *Runtime) storageToImage(storageImage *storage.Image, ref types.ImageRef
 	}
 }
 
+// storageToImageSlices transforms slice of storage.Image to slice of Image.
+func (r *Runtime) storageToImageSlices(storageImages []storage.Image, ref types.ImageReference) []*Image {
```

Comment:
The `ref` parameter is always `nil` (and it ~must be so, otherwise `StorageReference` would be the same, implying that the images are all “the same” in some sense).

---

Every caller does `MultiList`+`storageToImageSlices`, maybe this utility function can also do the `MultiList` part.